docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/MapValueToKeyMultiColumn.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class MapValueToKeyMultiColumn
+    {
+        /// This example demonstrates the use of the ValueMappingEstimator by mapping strings to other string values, or floats to strings. 
+        /// This is useful to map types to a grouping. 
+        /// It is possible to have multiple values map to the same category.
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Get a small dataset as an IEnumerable.
+            var rawData = new[] {
+                new DataPoint() { StudyTime = "0-4yrs" , DevelopmentTime = "6-11yrs" },
```

Comment:
>6-11yrs [](start = 76, length = 7)

can we use something other than time? Maybe CourseName? I don't want to give the impression that the values of the two columns should be similar or related in any way.

Reply:
i kept it to the same things because of the lookup map. It can have keys for two distinct categories, but i think if they they do multicolumn,  it will most likely be for two separate columns that have the same categories of values. 


---
In reply to: [271978738](https://github.com/dotnet/machinelearning/pull/3187#discussion_r271978738) [](ancestors = 271978738)